modules/gateway_module/tests/gateway/fnoc1.nc.nc.das.bescmd.baseline

```diff
@@ -28,9 +28,8 @@ Attributes {
     NC_GLOBAL {
         String base_time "88- 10-00:00:00";
         String title " FNOC UV wind components from 1988- 10 to 1988- 13.";
-        String history "removed date-timehyrax-1.16.4 http://test.opendap.org/opendap/data/nc/fnoc1.nc.nc
-";
-        String history_json "[{"$schema":"https:\/\/harmony.earthdata.nasa.gov\/schemas\/history\/0.1.0\/history-0.1.0.json","date_time":"removed date-time+0000","program":"hyrax","version":"1.16.4","parameters":[{"request_url":"http:\/\/test.opendap.org\/opendap\/data\/nc\/fnoc1.nc.nc"}]}]";
+        String history "removed date-timehyrax-1.16.4 http://test.opendap.org/opendap/data/nc/fnoc1.nc.nc\012";
```

Comment:
A question: Why there is  \012 added at the end of  new "history" attribute?

Reply:
We returned to escaping the character in the attributes. This was the behavior until 1.16.3 when we stopped escaping the attrs. We did that as part of the history_json work (although we abandoned the work in netcdf_handler and implemented history_json in a completely different way here) and was left in place by accident.

Reply:
Is this escaping feature also back to the HDF handlers? I also slacked you the same question.